### PR TITLE
Add deploy script

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #[issue number] by @[issue author]

## Description

This PR adds a script to copy built files from the `dist` folder to the `docs` folder, and adds the CNAME file to it so that the `docs` folder is ready for deployment on the github pages.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
This is fine, although the copy process could just as easily be accomplished with `rm` and `cp` and prevent us from adding a new package just for this purpose.

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "license": "MIT",
     "scripts": {
         "serve": "vue-cli-service serve",
-        "build": "vue-cli-service build",
+        "build": "vue-cli-service build && node deploy.js",
```
Comment (dhruvkb) [inline package.json:14]:
```suggestion
        "build": "vue-cli-service build && rm -r ./docs && cp -r ./dist ./docs",
```

**Review by dhruvkb — COMMENTED** [review]
Something like this:

File: package.json (hunk 1 above)
Comment (dhruvkb) [reply package.json:14]: And also
```
echo "<domain>" > CNAME
```

Comment (obulat) [reply package.json:14]: This looks much cleaner! It doesn't work on Windows, but I guess we don't have to build on Windows anyways. I will add the change once I test it on Ubuntu (WSL)

**Review by dhruvkb — COMMENTED** [review]
Why are there still changes to `package-lock.json`?

**Comment by obulat** [comment]
> Why are there still changes to `package-lock.json`?

I have deleted the `package-lock.json` and node_modules completely and then ran `npm install`. I wanted to make sure that  `package-lock.json` has the correct dependencies locked, and doesn't have outdated information.

**Review by dhruvkb — APPROVED** [review]
Cool, that makes sense. Approved 👍

**Pull request merged**